init/eessi_environment_variables

```diff
@@ -106,6 +106,15 @@ if [ -d $EESSI_PREFIX ]; then
             false
           fi
 
+        # Fix wrong path for RHEL >=8 libcurl 
+	# This is required here because we ship curl in our compat layer. If we only provided
+	# curl as a module file we could instead do this via a `modluafooter` in an EasyBuild
+	# hook (or via an Lmod hook)
+	rhel_libcurl_file="/etc/pki/tls/certs/ca-bundle.crt"
+        if [ -f $rhel_libcurl_file ]; then
+          show_msg "Found libcurl CAs file at RHEL location, setting CURL_CA_BUNDLE"
+          export CURL_CA_BUNDLE=$rhel_libcurl_file
+        fi
```

Comment:
```suggestion
          fi

          # Fix wrong path for RHEL >=8 libcurl 
	  # This is required here because we ship curl in our compat layer. If we only provided
	  # curl as a module file we could instead do this via a `modluafooter` in an EasyBuild
	  # hook (or via an Lmod hook)
	  rhel_libcurl_file="/etc/pki/tls/certs/ca-bundle.crt"
          if [ -f $rhel_libcurl_file ]; then
            show_msg "Found libcurl CAs file at RHEL location, setting CURL_CA_BUNDLE"
            export CURL_CA_BUNDLE=$rhel_libcurl_file
          fi
```
indentation was off